### ISSUE TITLE
An error occured while sending or receiving mail. SMTP server does not accept the STARTTLS command.

### ISSUE THREAD

**Issue by xwlee** [zetacomponents/Mail#34]
I try to send an email using SMTP with tls connection, but receive this error. 
FYI, I'm using Mailgun server.

```shell
telnet smtp.mailgun.org 587
Trying 50.56.21.178...
Connected to smtp.mailgun.org.
Escape character is '^]'.
220 ak47 ESMTP ready
ehlo blog.mailgun.com
250-ak47
250-AUTH PLAIN LOGIN
250-SIZE 52428800
250-8BITMIME
250-ENHANCEDSTATUSCODES
250 STARTTLS
```

After some checking, the cause is

```php
<?php 

...

if ( !preg_match( "/250-STARTTLS/", $response) )
{
    throw new ezcMailTransportSmtpException( 'SMTP server does not accept the STARTTLS command.' );
}
```

in https://github.com/zetacomponents/Mail/blob/master/src/transports/smtp/smtp_transport.php#L618

It match `250-STARTTLS` but not `250 STARTTLS`

**Comment by xwlee** [comment]
This issue is resolved by this commit f4cf1deaf41c704707eae5f46345736cf2611044

**Comment by derickr** [comment]
Oops - sorry I missed this one. Thanks for catching it!
